QUEST_LV_0100.tsv

```diff
@@ -7906,146 +7906,146 @@ QUEST_LV_0100_20150804_007901	$Historian Rexipher is thinking deeply about somet
-QUEST_LV_0100_20150908_007985	You can hear someone crying out!
-QUEST_LV_0100_20150908_007986	Passing the pot of the spirit
-QUEST_LV_0100_20150908_007987	The portal to the entrance to the royal tomb has been opened!
-QUEST_LV_0100_20150908_007988	Putting in the power
-QUEST_LV_0100_20150908_007989	Handing over the activating stones
-QUEST_LV_0100_20150908_007990	The guardian of the royal family treasure opened the mouth
-QUEST_LV_0100_20150908_007991	Destroy the polluted stone lantern!
-QUEST_LV_0100_20150908_007992	This isn't the only stone lantern{nl}Find the other polluted stone lanterns and destroy them
-QUEST_LV_0100_20150908_007993	Obtaining the power of the token
-QUEST_LV_0100_20150908_007994	The powerful aura of Wheelen is surrounding the body
-QUEST_LV_0100_20150908_007995	Defeat the guardian of the royal tomb who is attacking!
-QUEST_LV_0100_20150908_007996	It's a trap! Defeat the monsters that are rushing in!
-QUEST_LV_0100_20150908_007997	You didn't find anything!
-QUEST_LV_0100_20150908_007998	As you activated the device, the portal has opened!{nl}Check the location of the portal from the map!
-QUEST_LV_0100_20150908_007999	Follow Rexipher and go to the 3rd floor of the royal tomb!
+QUEST_LV_0100_20150908_007980	$You are enlightened after checking what was written on the tombstones of Lydia Schaffen{nl}You receive 1 status point!
+QUEST_LV_0100_20150908_007981	$Preparing a live sacrifice
+QUEST_LV_0100_20150908_007982	$Talking
+QUEST_LV_0100_20150908_007983	$Stop the Guardians from destroying the Royal Mausoleum!
+QUEST_LV_0100_20150908_007984	$It seems that the Royal Mausoleum will collapse soon!{nl}Stop the Guardians attacking it!
+QUEST_LV_0100_20150908_007985	$You can hear someone crying out!
+QUEST_LV_0100_20150908_007986	$Placing the Soul Jar
+QUEST_LV_0100_20150908_007987	$A portal to the entrance to the Royal Mausoleum has been opened!
+QUEST_LV_0100_20150908_007988	$Charging Power
+QUEST_LV_0100_20150908_007989	$Handing over the activating stones
+QUEST_LV_0100_20150908_007990	$The Guardian of the Royal Family Treasure opened its mouth
+QUEST_LV_0100_20150908_007991	$Destroy the corrupted stone lantern!
+QUEST_LV_0100_20150908_007992	$This isn't the only stone lantern{nl}Find the other corrupted stone lanterns and destroy them
+QUEST_LV_0100_20150908_007993	$Obtaining the marks' power
+QUEST_LV_0100_20150908_007994	$The powerful aura of Wheelen is surrounding your body
+QUEST_LV_0100_20150908_007995	$Defeat the Guardian of the Royal Mausoleum who is attacking!
+QUEST_LV_0100_20150908_007996	$It's a trap! Defeat the monsters that are rushing in!
+QUEST_LV_0100_20150908_007997	$You didn't find anything!
+QUEST_LV_0100_20150908_007998	$As you activated the device, the portal has opened!{nl}Check the location of the portal from the map!
+QUEST_LV_0100_20150908_007999	$Follow Rexipher and go to the 3rd floor of the Royal Mausoleum!
```

Comment:
Follow Rexipher and go to the 3rd floor of the Royal Mausoleum! -> Follow Rexipher and go to Royal Mausoleum 3F!